evap/contributor/tests/test_views.py

```diff
@@ -245,8 +245,6 @@ def test_contact_modal_escape(self):
         self.evaluation.save()
         page = self.app.get(self.url, user=self.responsible, status=200)
 
-        self.assertIn("changeParticipantRequestModalLabel", page)
-
```

Comment:
The changeEvaluationRequestModal still contains the evaluation title, so I think this should be replaced.